projects/igniteui-angular/src/lib/icon/icon.service.ts

```diff
@@ -1,6 +1,7 @@
-import { Injectable, SecurityContext, Inject } from '@angular/core';
+import { Injectable, SecurityContext, Inject, OnDestroy, Optional } from '@angular/core';
 import { DomSanitizer } from '@angular/platform-browser';
 import { DOCUMENT } from '@angular/common';
+import { HttpClient } from '@angular/common/http';
```

Comment:
There was an issue with importing `HttpClient` in the icon module in the past. Can you check what the issue was?

Reply:
I recall there were problems with http interceptors in the past. Especially if our icon module was lazy loaded.

Reply:
Reading the issue, it seems it only happens in JIT compiled apps?

Reply:
@rkaraivanov is right. It was something with HttpInterceptors, but honestly I don't remember what the issue was.